src/gui/advancedsettings.cpp

```diff
@@ -542,6 +545,12 @@ void AdvancedSettings::loadAdvancedSettings()
     m_spinBoxUPnPLeaseDuration.setSuffix(tr(" s", " seconds"));
     addRow(UPNP_LEASE_DURATION, (tr("UPnP lease duration [0: Permanent lease]") + ' ' + makeLink("https://www.libtorrent.org/reference-Settings.html#upnp_lease_duration", "(?)"))
         , &m_spinBoxUPnPLeaseDuration);
+    // Type of service
+    m_spinBoxPeerToS.setMinimum(0);
+    m_spinBoxPeerToS.setMaximum(255);
+    m_spinBoxPeerToS.setValue(session->PeerToS());
+    addRow(PEER_TOS, (tr("Type of service (ToS) to peers") + ' ' + makeLink("https://www.libtorrent.org/reference-Settings.html#peer_tos", "(?)"))
```

Comment:
Shouldn't it be "...for peers" or "...of peers"? Although, my English is bad, so I may be wrong here.

Reply:
It is rephrased.